shopfloor_mobile/static/wms/src/scenario/zone_picking.js

```diff
@@ -116,20 +116,31 @@ const template_mobile = `
             :card_color="utils.colors.color_for('screen_step_done')"
             />
         <item-detail-card
-            v-if="state_in(['set_line_destination', 'stock_issue', 'change_pack_lot'])"
+            v-if="state_in(['set_line_destination', 'stock_issue', 'change_pack_lot']) && !hide_qty_picker()"
             :key="make_state_component_key(['detail-move-line-product', state.data.move_line.id])"
             :record="state.data.move_line"
             :options="utils.wms.move_line_product_detail_options(state.data.move_line, {fields_blacklist: ['quantity']})"
             :card_color="utils.colors.color_for(state_in(['set_line_destination']) ? 'screen_step_done': 'screen_step_todo')"
             />
+
+
+        <item-detail-card
+            v-if="hide_qty_picker()"
+            :key="make_state_component_key(['detail-move-line-dest-pack', state.data.move_line.id])"
+            :record="state.data.move_line"
+            :options="{main: true, key_title: 'package_dest.name', title_action_field: {action_val_path: 'package_dest.name'}}"
+            :card_color="utils.colors.color_for(state_in(['set_line_destination']) ? 'screen_step_done': 'screen_step_todo')"
+            />
+
+
```

Comment:
What is the purpose of this item-detail-card?
With this item-detail-card the screen looks like this. 
![image](https://user-images.githubusercontent.com/78783475/234766636-5c76db4e-5226-4ac2-bdc5-2af9b00fe00f.png)
without it
![image](https://user-images.githubusercontent.com/78783475/234766826-73674f71-623f-4ccb-85b8-ba028ec107f9.png)
```suggestion
```

Reply:
ok the problem is something else :-)
```suggestion


        <item-detail-card
            v-if="hide_qty_picker()"
            :key="make_state_component_key(['detail-move-line-dest-pack', state.data.move_line.id])"
            :record="state.data.move_line"
            :options="{main: true, key_title: 'package_src.name', title_action_field: {action_val_path: 'package_src.name'}}"
            :card_color="utils.colors.color_for(state_in(['set_line_destination']) ? 'screen_step_done': 'screen_step_todo')"
            />


```